docker-compose-mysql.yml

```diff
@@ -1,85 +1,90 @@
-orderservice:
-  build: ./order-service/
-  ports:
-    - "8081:8080"
-  volumes:
-    - ./build/libs:/app
-  links:
-    - mysql
-    - kafka
-    - zookeeper
-    - cdcservice
-  environment:
-    SPRING_DATASOURCE_URL: jdbc:mysql://mysql/eventuate
-    SPRING_DATASOURCE_USERNAME: mysqluser
-    SPRING_DATASOURCE_PASSWORD: mysqlpw
-    SPRING_DATASOURCE_DRIVER_CLASS_NAME: com.mysql.jdbc.Driver
-    EVENTUATELOCAL_KAFKA_BOOTSTRAP_SERVERS: kafka:9092
-    EVENTUATELOCAL_ZOOKEEPER_CONNECTION_STRING: zookeeper:2181
+version: '3'
+services:
+  orderservice:
+    build: ./order-service/
+    ports:
+      - "8081:8080"
+    volumes:
+      - ./build/libs:/app
+    depends_on:
+      - mysql
+      - kafka
+      - zookeeper
+      - cdcservice
+    environment:
+      SPRING_DATASOURCE_URL: jdbc:mysql://mysql/eventuate
+      SPRING_DATASOURCE_USERNAME: mysqluser
+      SPRING_DATASOURCE_PASSWORD: mysqlpw
+      SPRING_DATASOURCE_DRIVER_CLASS_NAME: com.mysql.jdbc.Driver
+      EVENTUATELOCAL_KAFKA_BOOTSTRAP_SERVERS: kafka:9092
+      EVENTUATELOCAL_ZOOKEEPER_CONNECTION_STRING: zookeeper:2181
 
-customerservice:
-  build: ./customer-service/
-  ports:
-    - "8082:8080"
-  volumes:
-    - ./build/libs:/app
-  links:
-    - mysql
-    - kafka
-    - zookeeper
-    - cdcservice
-  environment:
-    SPRING_DATASOURCE_URL: jdbc:mysql://mysql/eventuate
-    SPRING_DATASOURCE_USERNAME: mysqluser
-    SPRING_DATASOURCE_PASSWORD: mysqlpw
-    SPRING_DATASOURCE_DRIVER_CLASS_NAME: com.mysql.jdbc.Driver
-    EVENTUATELOCAL_KAFKA_BOOTSTRAP_SERVERS: kafka:9092
-    EVENTUATELOCAL_ZOOKEEPER_CONNECTION_STRING: zookeeper:2181
+  customerservice:
+    build: ./customer-service/
+    ports:
+      - "8082:8080"
+    volumes:
```

Comment:
I don't think `volumes:` is used any more in any project.
Please review.

Reply:
I will check